corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail.xml

```diff
@@ -31,7 +31,7 @@
         </header>
         <template>
           <text>
-            <xpath function="description"/>
+            <xpath function="report description"/>
```

Comment:
Yeah, I agree with waiting on feedback, and that we could set up some checkbox-button-whatever to force the user to explicitly choose between xpath and localized string.